<?xml version="1.0" encoding="UTF-8" ?>
<testsuites disabled="0" errors="0" failures="1" tests="2" time="6.333">
 <testsuite name="demo calculator tests" timestamp="2023-09-12T21:56:40" hostname="localhost" time="6.333" errors="0" tests="2" skipped="0" disabled="0" failures="1">
  <testcase classname="demo calculator tests" name="addition test" time="5.974" />
  <testcase classname="demo calculator tests" name="subtraction test" time="0.336">
   <failure type="exception" message="Failed: homepage.clickGoButtonError is not a function"><![CDATA[TypeError: homepage.clickGoButtonError is not a function
    at UserContext.<anonymous> (C:\Users\JOrbiso\Documents\repos\Protractor\tests\calculator.js:28:18)
    at C:\Users\JOrbiso\AppData\Roaming\npm\node_modules\protractor\node_modules\jasminewd2\index.js:112:25
    at new ManagedPromise (C:\Users\JOrbiso\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:1077:7)
    at ControlFlow.promise (C:\Users\JOrbiso\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:2505:12)
    at schedulerExecute (C:\Users\JOrbiso\AppData\Roaming\npm\node_modules\protractor\node_modules\jasminewd2\index.js:95:18)
    at TaskQueue.execute_ (C:\Users\JOrbiso\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:3084:14)
    at TaskQueue.executeNext_ (C:\Users\JOrbiso\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:3067:27)
    at C:\Users\JOrbiso\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:2927:27
    at C:\Users\JOrbiso\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:668:7
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
From: Task: Run it("subtraction test") in control flow
    at UserContext.<anonymous> (C:\Users\JOrbiso\AppData\Roaming\npm\node_modules\protractor\node_modules\jasminewd2\index.js:94:19)
    at attempt (C:\Users\JOrbiso\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4297:26)
    at QueueRunner.run (C:\Users\JOrbiso\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4217:20)
    at QueueRunner.execute (C:\Users\JOrbiso\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4199:10)
    at Spec.queueRunnerFactory (C:\Users\JOrbiso\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:909:35)
    at Spec.execute (C:\Users\JOrbiso\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:526:10)
    at UserContext.fn (C:\Users\JOrbiso\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:5340:37)
    at attempt (C:\Users\JOrbiso\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4297:26)
    at QueueRunner.run (C:\Users\JOrbiso\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4217:20)
    at runNext (C:\Users\JOrbiso\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4257:20)
From asynchronous test: 
Error
    at Suite.<anonymous> (C:\Users\JOrbiso\Documents\repos\Protractor\tests\calculator.js:20:5)
    at addSpecsToSuite (C:\Users\JOrbiso\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:1107:25)
    at Env.describe (C:\Users\JOrbiso\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:1074:7)
    at describe (C:\Users\JOrbiso\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4399:18)
    at Object.<anonymous> (C:\Users\JOrbiso\Documents\repos\Protractor\tests\calculator.js:3:1)
    at Module._compile (node:internal/modules/cjs/loader:1256:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1310:10)
    at Module.load (node:internal/modules/cjs/loader:1119:32)
    at Function.Module._load (node:internal/modules/cjs/loader:960:12)]]>
   </failure>
  </testcase>
 </testsuite>
</testsuites>